### PR TITLE
Feature/get user

### PR DESCRIPTION
**Pull request by mwnicks**
### What

Added the get user endpoint

### How to review

Ensure all unit and feature tests are passing

Send a GET request to /v1/users/ae54bd71-6099-4364-acd3-519a64202679
You should receive a 200 response with the following JSON body
{
    "forename": "Matt",
    "lastname": "Nicks",
    "email": "matt.nicks@ons.gov.uk",
    "groups": [],
    "status": "CONFIRMED",
    "id": "ae54bd71-6099-4364-acd3-519a64202679"
}

Send a GET request to /v1/users/invalid-id
You should receive a 404 response with a user not found error in the body.

### Who can review

Anyone but me

### CODE REVIEW

File: api/users.go [hunk 1]
```diff
@@ -86,6 +87,31 @@ func (api *API) ListUsersHandler(ctx context.Context, w http.ResponseWriter, req
 	return models.NewSuccessResponse(jsonResponse, http.StatusOK, nil), nil
 }
 
+//GetUserHandler lists the users in the user pool
+func (api *API) GetUserHandler(ctx context.Context, w http.ResponseWriter, req *http.Request) (*models.SuccessResponse, *models.ErrorResponse) {
+	vars := mux.Vars(req)
+	user := models.UserParams{ID: vars["id"]}
+	getUserInput := user.BuildAdminGetUserRequest(api.UserPoolId)
```
Comment (justinpjose) [inline api/users.go]: The variables names `getUserInput` and `getUserResp` can be `userInput` and `userResp` respectively. Functions and methods should be named with a _verb_ while variables should be named with a _noun_.

https://softwareengineering.stackexchange.com/questions/33532/naming-conventions-for-variables

**Review by justinpjose — COMMENTED** [review]
Just one small comment. Otherwise, this looks all good. The tests all pass as well (both unit and feature tests work)

File: api/users.go (hunk 1 above)
Comment (mwnicks) [reply api/users.go]: I've updated the variable names, thanks for the guidance on naming conventions.

**Review by HeJones — APPROVED** [review]
The expected behaviour was seen.

**Pull request merged**